packages/dashboard/src/pages/CreateMaskWallet/CreateMnemonic/index.tsx

```diff
@@ -0,0 +1,446 @@
+        const t = useDashboardI18N()
+        const { classes, cx } = useStyles({ isVerify: true })
+
+        return (
+            <>
+                <Typography className={cx(classes.title, classes.helveticaBold)}>
+                    {t.wallets_create_wallet_verification()}
+                </Typography>
+                <Typography className={classes.tips}>{t.create_wallet_verify_words()}</Typography>
+                <Box component="ul" className={classes.puzzleWordList}>
+                    {puzzleWordList.map((puzzleWord, index) => (
+                        <section key={index} className={classes.puzzleWord}>
+                            <Typography className={classes.puzzleWordIndex}>{puzzleWord.index + 1}.</Typography>
+                            <PuzzleOption
+                                puzzleWord={puzzleWord}
+                                answerCallback={answerCallback}
+                                puzzleAnswer={puzzleAnswer}
+                            />
+                        </section>
+                    ))}
+                </Box>
+                {isMatched === false ? (
+                    <Typography className={classes.verificationFail}>
+                        {t.create_wallet_mnemonic_verification_fail()}
+                    </Typography>
+                ) : null}
+                <SetupFrameController>
+                    <div className={classes.buttonGroup}>
+                        <SecondaryButton
+                            className={classes.helveticaBold}
+                            width="125px"
+                            size="large"
+                            onClick={() => setIsVerify(false)}>
+                            {t.back()}
+                        </SecondaryButton>
+                        <PrimaryButton
+                            className={classes.helveticaBold}
+                            width="125px"
+                            size="large"
+                            color="primary"
+                            onClick={() => verifyAnswerCallback(onSubmit)}>
+                            {t.verify()}
+                        </PrimaryButton>
+                    </div>
+                </SetupFrameController>
+            </>
+        )
+    },
+)
+
+const PuzzleOption = memo<PuzzleOption>(({ puzzleWord, puzzleAnswer, answerCallback }) => {
+    const { classes, cx } = useStyles({ isVerify: false })
+
+    return (
+        <>
+            {puzzleWord.options.map((word, index) => (
+                <section
+                    key={index}
+                    className={classes.puzzleOption}
+                    onClick={() => answerCallback(puzzleWord.index, word)}>
+                    <div
+                        className={cx(
+                            classes.checkIconWrapper,
+                            word === puzzleAnswer[puzzleWord.index] ? classes.checked : '',
+                        )}>
+                        <CheckIcon
```

Comment:
Avoid using external icons, we should plan to get rid of them.
Use icon from our icon collection, if there is no matching icon, ask designer to add one.

Reply:
done